requirements.txt

```diff
@@ -39,3 +39,5 @@ zstandard; python_version >= '3.5'
 pyyaml; python_version == '2.7' or python_version >= '3.6'
 
 Jinja2 >= 2.10.2
+
```

Comment:
I put here only stuff absolutely necessary for compiling, so I would remove that here.